consensus/broadcast/delayedBroadcast_test.go

```diff
@@ -37,6 +40,19 @@ type validatorDelayArgs struct {
 	order            uint32
 }
 
+type syncLogObserver struct {
+	buffer    *bytes.Buffer
+	mutex     *sync.Mutex
+	formatter logger.Formatter
```

Comment:
formatter not needed in this struct

also `mutex *sync.Mutex` can be simplified here to `sync.Mutex`, thus `observer.Lock()` should be used in tests

Reply:
perhaps add a new method getBufferStr() string and reuse it:
```go
func (o *syncLogObserver) getBufferStr() string {
	o.Lock()
	logOutputStr := o.buffer.String()
	o.Unlock()
	
	return logOutputStr
}
```